engine/api/src/main/scala/pl/touk/nussknacker/engine/defaults/deployment/DefaultStateStatus.scala

```diff
@@ -0,0 +1,17 @@
+package pl.touk.nussknacker.engine.defaults.deployment
+
+import pl.touk.nussknacker.engine.api.deployment.StatusState
+
+object DefaultStateStatus extends Enumeration {
```

Comment:
Is it really global defaults? Maybe better would be FlinkStateStatus or at least StreamingStateStatus?

Reply:
Hmm.. I can do StreamingStateStatus and StandaloneStateStatus.

Reply:
I've changed it to SimpleStateStatus. And I've crated FlinkStateStatus.